java/core/src/test/java/com/microsoft/bond/testmodel/A.java

```diff
@@ -285,6 +284,11 @@ protected final void deserializeStructFields(TaggedDeserializationContext contex
         }
 
         @Override
+        protected final void deserializeStructFields(UntaggedDeserializationContext context, A<X, Y> value) throws IOException {
+        }
```

Comment:
Can you please delete them, if you don't want to maintain them?

---
In reply to: [131010024](https://github.com/Microsoft/bond/pull/542#discussion_r131010024) [](ancestors = 131010024)